UDV-Core/src/Modules/Temporal/Utils/VersionsManager.js

```diff
@@ -0,0 +1,32 @@
+import { $3DTemporalVersion } from '../3DTILES_temporal/3DTemporalVersion';
+
+/**
+Manage the version found in the tileset.json
+Only do parsing
+*/
+export class VersionsManager {
```

Comment:
This class is useless since the constructor calls the only method of the class which fills a list with the content of another list without doing any operation on it.

Reply:
Cleaned